app/src/main/java/eu/kanade/tachiyomi/ui/reader/ReaderActivity.kt

```diff
@@ -1526,11 +1527,13 @@ class ReaderActivity : BaseRxActivity<ReaderPresenter>() {
 
     private fun openMangaInBrowser() {
         val source = presenter.getSource() ?: return
-        val url = try {
+        val chapterUrl = presenter.getCurrentChapter()?.chapter?.url?.getUrlWithoutDomain()
```

Comment:
the reason this wasnt implemented before is because the chapter url used for tachi and the chapter url used for webview are different urls. For example mangadex extensions uses "api.mangadex.com" while the webpage is just "mangadex.com"

Reply:
Yes that's why I did it like this: baseurl + chapterUrlWithoutDomain. I use more than 30 sources and so far every single one of them works (even mangadex) but you are right some of them might not work as expected

Reply:
I guess leave it for now then, someone else will bring it up on a source that doesn't work

Instead I'll ask can you add this logic to `ReaderActivity.onProvideAssistContent` as well?